### PR TITLE
[enhancement](proc) Support showing more details in show proc "/dbs"

### LINKED ISSUE #10470: [Enhancement] (proc) Support showing more details in show proc "/dbs"
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

now, we need the global view about db used quota. so, I add more details in show proc "/dbs"

### Solution

```
mysql> show proc "/dbs";
+-------+------------------------------------+----------+--------+-------------+--------------------------+--------------+--------------+
| DbId  | DbName                             | TableNum | Size   | Quota       | LastConsistencyCheckTime | ReplicaCount | ReplicaQuota |
+-------+------------------------------------+----------+--------+-------------+--------------------------+--------------+--------------+
| 0     | default_cluster:information_schema | 22       | 0.000  | 1024.000 TB | NULL                     | 0            | 1073741824   |
| 11001 | default_cluster:test               | 0        | 0.000  | 1024.000 TB | NULL                     | 0            | 1073741824   |
+-------+------------------------------------+----------+--------+-------------+--------------------------+--------------+--------------+
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by caiconghui**
# Proposed changes

Issue Number: close #10470 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java [hunk 1]
```diff
@@ -90,33 +88,31 @@ public ProcResult fetchResult() throws AnalysisException {
...
-                long replicaQuota = 0;
-                if (db instanceof Database) {
-                    long dataQuota = ((Database) db).getDataQuota();
-                    Pair<Double, String> quotaUnitPair = DebugUtil.getByteUint(dataQuota);
-                    readableQuota =
-                            DebugUtil.DECIMAL_FORMAT_SCALE_3.format(quotaUnitPair.first) + " " + quotaUnitPair.second;
-                    lastCheckTime = TimeUtils.longToTimeString(((Database) db).getLastCheckTime());
-                    replicaQuota = ((Database) db).getReplicaQuota();
-                }
+                long usedDataQuota = ((Database) db).getUsedDataQuotaWithLock();
```
Comment (stalary) [inline fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java:105]: Would it be better to declare a `Database = (Database) db` first？

Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java:105]: I think both are ok. just keep same style with origin code.

File: fe/fe-core/src/test/java/org/apache/doris/common/proc/DbsProcDirTest.java [hunk 2]
```diff
@@ -192,11 +191,12 @@ public void testFetchResultNormal() throws AnalysisException {
...
         Assert.assertTrue(result instanceof BaseProcResult);
 
-        Assert.assertEquals(Lists.newArrayList("DbId", "DbName", "TableNum", "Quota", "LastConsistencyCheckTime", "ReplicaQuota"),
+        Assert.assertEquals(Lists.newArrayList("DbId", "DbName", "TableNum", "Size", "Quota",
+                    "LastConsistencyCheckTime", "ReplicaCount", "ReplicaQuota"),
                 result.getColumnNames());
         List<List<String>> rows = Lists.newArrayList();
-        rows.add(Arrays.asList(String.valueOf(db1.getId()), db1.getFullName(), "0", "1024.000 TB", FeConstants.null_string, "1073741824"));
-        rows.add(Arrays.asList(String.valueOf(db2.getId()), db2.getFullName(), "0", "1024.000 TB", FeConstants.null_string, "1073741824"));
+        rows.add(Arrays.asList(String.valueOf(db1.getId()), db1.getFullName(), "0", "0.000 ", "1024.000 TB", "\\N", "0", "1073741824"));
```
[the remark this answers is not in the document]
Comment (caiconghui) [reply fe/fe-core/src/test/java/org/apache/doris/common/proc/DbsProcDirTest.java]: fix

File: fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java (hunk 1 above)
Comment (stalary) [reply fe/fe-core/src/main/java/org/apache/doris/common/proc/DbsProcDir.java:105]: Ok, no problem~

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**